src/pkg/utils/utils.go

```diff
@@ -90,6 +90,7 @@ var (
 		"HISTIGNORE",
 		"HISTSIZE",
 		"HISTTIMEFORMAT",
+		"KONSOLE_VERSION",
```

Comment:
It will be good to add some tests for `KONSOLE_VERSION` to `test/system/220-environment-variables.bats`.

Reply:
I have added some tests. I had some difficulties running them locally, but the bot seems to be happy.

![image](https://github.com/containers/toolbox/assets/138280734/4b6923a1-d328-49a5-bb70-86672428496f)
[system-test-fedora-39/console](https://softwarefactory-project.io/zuul/t/local/build/334f5a2571684213a9bc6e5f7ddf93d7/console)